src/_guides/libraries/useful-libraries.md

```diff
@@ -0,0 +1,147 @@
+---
+layout: default
+title: "Commonly Used Dart Libraries"
+description: "What are some of the most useful and popular Dart libraries and where can you learn more?"
+---
+
+You can use many Dart libraries when writing your code. You can find
+Dart libraries in three ways:
+
+* The core libraries&mdash;such as dart:core, dart:async, and dart:collection&mdash;are
+  distributed with the SDK and documented at [api.dartlang.org]({{site.dart_api}}).
+* Libraries shared with the Dart community are distributed as library packages,
+  published at [pub.dartlang.org](https://pub.dartlang.org/).
+  The [pub](/tools/pub/) tool allows you to create, publish, and manage library packages.
+* You can use libraries from GitHub or your local file system by placing them in the
```

Comment:
Let's make this parallel to the previous two items (even if it makes it passive voice) by not making it start with "you".

We should also split this in two, because you don't have to place GitHub libraries in your /lib directory (and github isn't the only valid source). So the 4 ways are:
* core
* pub.dartlang.org
* under lib/
* something else (a URL, git, SDK, path, ... point them to https://www.dartlang.org/tools/pub/dependencies for details)